src/api/transaction/suspay-finish.js

```diff
@@ -0,0 +1,48 @@
+/* @flow */
+'use strict';
+const _ = require('lodash');
+const utils = require('./utils');
+const validate = utils.common.validate;
+const Transaction = utils.common.core.Transaction;
+
+function createSusPayFinishTransaction(account, payment) {
+  validate.address(account);
+  validate.suspendedPaymentFinish(payment);
+
+  const transaction = new Transaction();
+  transaction.susPayFinish({
+    from: account,
+    owner: payment.owner,
+    offerSequence: payment.offerSequence
```

Comment:
I don't think we should be letting the implementation details bubble up such that we have a transaction field with a misleading name. I think we should just add the new generic field if these SusPay critters have nothing to do with offers. 

Ashame we have bollix fields like this:
```
    WalletSize(12, Type.UInt32),
```

Probably TargetSeq should be TargetSequence for consistency with OfferSequence, LastLedgerSequence etc?